extensions/awt/deployment/src/main/java/io/quarkus/awt/deployment/AwtProcessor.java

```diff
@@ -56,20 +293,23 @@ JniRuntimeAccessBuildItem setupJava2DClasses() {
                 "sun.java2d.loops.FillRect",
                 "sun.java2d.loops.FillSpans",
                 "sun.java2d.loops.GraphicsPrimitive",
-                "sun.java2d.loops.GraphicsPrimitiveMgr",
                 "sun.java2d.loops.GraphicsPrimitive[]",
+                "sun.java2d.loops.GraphicsPrimitiveMgr",
                 "sun.java2d.loops.MaskBlit",
                 "sun.java2d.loops.MaskFill",
                 "sun.java2d.loops.ScaledBlit",
                 "sun.java2d.loops.SurfaceType",
                 "sun.java2d.loops.TransformHelper",
                 "sun.java2d.loops.XORComposite",
+                "sun.java2d.NullSurfaceData",
+                "sun.java2d.pipe.BufferedMaskBlit",
+                "sun.java2d.pipe.GlyphListPipe",
                 "sun.java2d.pipe.Region",
-                "sun.java2d.pipe.RegionIterator");
-    }
-
-    @BuildStep
-    FeatureBuildItem feature() {
-        return new FeatureBuildItem(FEATURE);
+                "sun.java2d.pipe.RegionIterator",
+                "sun.java2d.pipe.ShapeSpanIterator",
+                "sun.java2d.pipe.SpanClipRenderer",
+                "sun.java2d.pipe.ValidatePipe",
+                "sun.java2d.SunGraphics2D",
+                "sun.java2d.SurfaceData");
```

Comment:
Wow, That's a huge list! Did you some kind of tool to generate it?

Reply:
@cescoffier I used a bio-robot.  JDK 11 source code in one terminal, JDK 17 source code in another terminal, Java2D/ImageIO/AWT pet projects and examples to touch relevant code paths.
That created the testing corpus. Then records to AwtProcessor were added one by one or in small couples/handfuls as it made sense according to the JDK source code.

The extension is done safely, so unless your application does something rather exotic, you are covered.

In future, especially due to fontconfig/freetype, I might make it configurable so as experienced users can switch something off, e.g. to keep processing TIFF/PNG/JPEG/GIF/BMP/WBMP files, but having nothing to do with fonts.
Opinions differ though, as it would add more knobs and buttons to the Quarkus config and it would make the integration more fragile. Not part of this PR I would say.